### ISSUE TITLE
Incoming token send not appearing

### ISSUE THREAD

**Issue by mr-michael** [tallycash/extension#642]
tl;dr I sent myself $10 or Orchid. It appears in etherscan (https://etherscan.io/address/0x1cbb77b65b07403ad7f85ff073bfc29a8212219c#tokentxns) and inside MM, but not in Tally (even after 15mins+)

<img width="1610" alt="Screen Shot 2021-12-14 at 10 06 36 PM" src="https://user-images.githubusercontent.com/7005061/146117190-b3e47b55-c05a-48e8-a47e-44a6bce09af5.png">

<img width="351" alt="Screen Shot 2021-12-14 at 10 19 38 PM" src="https://user-images.githubusercontent.com/7005061/146117196-f08bfd56-09f5-427f-93b4-02590ab04ce0.png">

<img width="387" alt="Screen Shot 2021-12-14 at 10 06 47 PM" src="https://user-images.githubusercontent.com/7005061/146117205-ee36f001-123a-469c-bf50-c527a13e6dd3.png">

**Comment by henryboldi** [comment]
Duplicate of #619
